src/bin/pg_autoctl/cli_service.c

```diff
@@ -432,25 +432,24 @@ cli_service_status(int argc, char **argv)
 					  pgSetup->pgdata, pgSetup->pidFile.pid);
 		}
 
+		/*
+		 * That's not supposed to happen, because pg_autoctl should be the
+		 * parent process of Postgres. That said, when in maintenance,
+		 * operators could stop pg_autoctl and then start/stop Postgres to make
+		 * some configuration changes, and then use pg_autoctl again.
```

Comment:
I guess this should be for the log above?

Reply:
Ah yeah would be clearer that way, done now.